.travis.yml

```diff
@@ -13,7 +13,35 @@ script:
   # run integration tests using an addons-linter binary in a production-like environment
   - npm run test-integration:production
   - npm run prettier-ci
```

Comment:
I think `prettier-ci` could also run in its own job and maybe some other scripts, too. WDYT?

Reply:
@willdurand I was thinking about splitting those too but I'm a bit on the fence about it: splitting the jobs would still introduce some additional time due to:
- some lag time on starting a sub-job (likely because the job is going to wait to get a worker node assigned)
- some additional time needed to clone the repo, install node using nvm and install all the dependencies with npm ci

and so opted to only split two sub-jobs (one for the web-ext functional tests and one for the deploy) and leave everything else in the main test sub-job.

what do you think? would it be worth to split prettier-ci in its own job from your perspective?

Reply:
I like to have one job per tool because in most cases, the test suite will take longer than other jobs and jobs run in parallel. I am fine with the current approach, though, it's more a matter of personal preference (I find it easier to see what failed without necessarily reading the logs every time).